apis/openstack-nova/src/test/java/org/jclouds/openstack/nova/v2_0/compute/NovaComputeServiceLiveTest.java

```diff
@@ -51,7 +78,66 @@ protected LoggingModule getLoggingModule() {
 
    @Override
    protected Iterable<Module> setupModules() {
-      return ImmutableSet.of(getLoggingModule(), credentialStoreModule, getSshModule());
+      return ImmutableSet.<Module> of(
+              ContextLinking.linkContext(neutronApiContext),
+              getLoggingModule(),
+              credentialStoreModule,
+              getSshModule()
+      );
+   }
```

Comment:
With this config we will *always* test nova+neutron, but not the path of only Nova. I think we should be able to still live test the old path for legacy environments.